plugins/obs-webrtc/whip-output.cpp

```diff
@@ -321,6 +363,28 @@ void WHIPOutput::ParseLinkHeader(std::string val, std::vector<rtc::IceServer> &i
 	}
 }
 
+size_t WHIPOutput::CountSimulcastLayers(std::string answer)
```

Comment:
Could probably move this out into a static method since it's just works on its own and doesn't use any class state.

Reply:
Fixed. This is now static

Reply:
Might be worth pulling into the utils and fixing up the name to reflect that this is parsing the response from the server. The current naming would have me think it's getting the layers from the output.

Reply:
👍 moved into whip-utils.